data-sources/amazon-verified-access/config.yml

```diff
@@ -0,0 +1,8 @@
+id: amazon-verified-access
+displayName: Amazon Verified Access
+description: Monitor Amazon Verified Access by sending Access logs to New Relic
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-verified-access-monitoring-integration/
```

Comment:
Not required, but worth considering here adding the 'aws' categoryTerm

```yml
categoryTerms:
  - aws
```

This makes sure the data source shows up in marketplace under the AWS category

[Category term mappings](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)

Reply:
Note also, we are waiting for the docs publication as well before we merge